src/Grpc.Net.Client/GrpcClient.cs

```diff
@@ -39,7 +39,33 @@ public static class GrpcClient
         /// <returns>A gRPC client.</returns>
         public static TClient Create<TClient>(HttpClient httpClient, ILoggerFactory? loggerFactory = null) where TClient : ClientBase
```

Comment:
Remove?

Reply:
This would be slightly more concise right? I don't mind this syntax sugar.

Reply:
Fair point in terms of simplicity. Let's remove